app/assets/javascripts/modules/map.js.coffee

```diff
@@ -17,8 +17,11 @@ window.Map = class Map
       name: 'Other'
 
   constructor: (elementId, mapOpts={}) ->
-    @baseMap = L.tileLayer('http://tile.cloudmade.com/BC9A493B41014CAABB98F0471D759707/997/256/{z}/{x}/{y}.png', {maxZoom: 17})
-    @baseSatellite =  new L.BingLayer("ApZALeudlU-OTm7Me2qekFHrstBXNdv3hft6qy3ZeTQWD6a460-QqCQyYnDigINc", {type: "Aerial", maxZoom: 19})
+    @baseMap = L.tileLayer(
+      'https://tile.openstreetmap.org/{z}/{x}/{y}.png', 
+      { maxZoom: 17}
+    )
+    @baseSatellite =  new L.BingLayer("ApZALeudlU-OTm7Me2qekFHrstBXNdv3hft6qy3ZeTQWD6a460-QqCQyYnDigINc", {type: "AerialWithLabels", maxZoom: 19})
```

Comment:
@WKocur I don't wish to be a pain - but this looks like an API key? I wondered if it was and whether we would want to use an ENV var for that rather than hardcode it?

Reply:
Happy to merge otherwise! We can always make another PR for the API key?